python/paddle/fluid/reader.py

```diff
@@ -414,8 +501,12 @@ def _init_iterable(self):
 
     def _start(self):
         if self._use_multiprocess:
+            self._thread_done_event = threading.Event()
             # Set data_queue and process
             self._data_queue = multiprocessing.Queue(self._capacity)
+            # add _data_queue into global queue set
+            global multiprocess_queue_set
+            multiprocess_queue_set.add(self._data_queue)
```

Comment:
Seems that `multiprocess_queue_set` would increase as epoch increase. Try to remove the original `self._data_queue` and pop all data outs, and then create a new `multiprocess.Queue`?

Reply:
yes, done